web-components/src/[sandbox]/sandbox.ts

```diff
@@ -213,6 +214,13 @@ export class Sandbox extends LitElement {
             </sass-stats>
           </div>
 
+          <div class="container" aria-label="md-colors">
+            <h2>md-colors</h2>
+            <sass-stats component="table">
```

Comment:
do we really need sass stats for this?

Reply:
Or at least add it to another folder (e.g. Internal/Not Exportable...) or give the folder a prefix... something

Reply:
Nope